pyproject.toml

```
@@ -96,6 +97,7 @@ filterwarnings = [
     "ignore::sklearn.exceptions.InconsistentVersionWarning:sklearn:",
     "ignore:.*invalid value encountered in cast.*:RuntimeWarning:numpy.*",
     "ignore:.*The abstract Provider and ProviderV1 classes are deprecated.*:DeprecationWarning:qiskit.*",
+    "ignore:.*qiskit.providers.models is deprecated since Qiskit 1.2 and will be removed in Qiskit 2.0.*:DeprecationWarning:qiskit.*",
```

```suggestion
    "ignore:.*qiskit\.providers\.models is deprecated.*:DeprecationWarning:qiskit.*",
```
You need to escape the dots here.